lib/services/ios-project-service.ts

```diff
@@ -221,11 +219,58 @@ class IOSProjectService implements  IPlatformProjectService {
 			shell.cp("-R", path.join(cachedPackagePath, "*"), path.join(this.platformData.projectRoot, util.format("%s.xcodeproj", this.$projectData.projectName)));
 			this.$logger.info("Copied from %s at %s.", cachedPackagePath, this.platformData.projectRoot);
 
-
 			var pbxprojFilePath = path.join(this.platformData.projectRoot, this.$projectData.projectName + IOSProjectService.XCODE_PROJECT_EXT_NAME, "project.pbxproj");
 			this.replaceFileContent(pbxprojFilePath).wait();
 		}).future<void>()();
 	}
+	
+	public prepareProject(): IFuture<void> {
+		return (() => {
+			let project = this.xcodeproj;
+			let resources = project.pbxGroupByName("Resources");
+			let references = project.pbxFileReferenceSection();
+			
+			let images = _(<any[]>resources.children)
+				.map(resource => references[resource.value])
+				.value();
+			
+			let imageNames = _.map(images, image => image.name.replace(/\"/g, ""));
```

Comment:
I think this variable is poorly named. the log.trace line bellow suggests a better name: xcodeProjectImages